static/script-tests/tests/devices/mediaplayer/cehtmlcommontests.js

```diff
@@ -346,6 +346,20 @@ window.commonTests.mediaPlayer.cehtml.mixinTests = function (testCase, mediaPlay
         });
     };
 
+    mixins.testBeginPlaybackFromSetsCurrentTimeAndCallsPlayOnMediaElementWhenInStoppedState = function(queue) {
```

Comment:
I was wondering about whether there should be tests around that, but I couldn't see any tests around calling in an invalid state for any other functions.